Maple2.Server.Game/Scripting/Npc/NpcScript.cs

```diff
@@ -13,6 +13,7 @@ public class NpcScript {
     }
 
     public bool Begin() {
+        context.SetTalkTypeFlags(script.first());
```

Comment:
Can we move this into the constructor so TalkType can remain `readonly`?

Reply:
can't make it `readonly` anyways, but I think it's still worth moving to the constructor.